Cleanup for about fluentbit and fluentd (#1455)
fluent/fluent-bit-docs: about/fluentd-and-fluent-bit.md

---
description: The production grade telemetry ecosystem
---

# Fluentd and Fluent Bit

Telemetry data processing can be complex, especially at scale. That's why
[Fluentd](https://www.fluentd.org) was born. Fluentd is more than a simple tool,
it's grown into a fullscale ecosystem that contains SDKs for different languages
and subprojects like [Fluent Bit](https://fluentbit.io).

Here, we describe the relationship between the [Fluentd](http://fluentd.org)
and [Fluent Bit](http://fluentbit.io) open source projects.

Both projects are:

- Licensed under the terms of Apache License v2.0.
- Graduated hosted projects by the [Cloud Native Computing Foundation (CNCF)](https://cncf.io).

> - Production grade solutions: Deployed **million** of times every single day.

Author: except the emphasis also should be removed heh

> - Production grade solutions: Deployed **million** of times every single day.

Author: ```suggestion
- Production grade solutions: Deployed millions of times every single day.
```

- Vendor neutral and community driven.
- Widely adopted by the industry: Trusted by major companies like AWS, Microsoft,

>   Google Cloud and hundreds of others.

Reviewer: ```suggestion
  Google Cloud, and hundreds of others.
```

The projects have many similarities: [Fluent Bit](https://fluentbit.io) is
designed and built on top of the best ideas of [Fluentd](https://www.fluentd.org)

> architecture and general design. Choosing one depends on end-user needs.

Reviewer: ```suggestion
architecture and general design. Which one you choose depends on your end-users' needs.
```

The following table describes a comparison of different areas of the projects:

> |              | Fluentd               | Fluent Bit            |

Reviewer: We should have a title for this column. "Attribute", perhaps?

Author: in theory, but I thought that might be confusing with the other two columns being products and also whether that was out of scope for general cleanup.

| ------------ | --------------------- | --------------------- |
| Scope        | Containers / Servers  | Embedded Linux / Containers / Servers |
| Language     | C & Ruby              | C                                     |

> | Memory       |  > 60MB               | ~1MB                                  |

Reviewer: ```suggestion
| Memory       |  Greater than 60&nbsp;MB | Approximately 1&nbsp;MB |
```

| Performance  | Medium Performance    | High Performance                      |
| Dependencies | Built as a Ruby Gem, depends on other gems. | Zero dependencies, unless required by a plugin. |

> | Plugins      | Over 1000 external plugins available. | Over 100 built-in plugins available. |

Reviewer: ```suggestion
| Plugins      | Over 1,000 external plugins available. | Over 100 built-in plugins available. |
```
